lib/geordi/commands/drop_databases.rb

```diff
@@ -19,6 +22,8 @@
   desc: 'Use Postgres port or socket'
 option :mysql, banner: 'PORT_OR_SOCKET',
   desc: 'Use MySQL/MariaDB port or socket'
+option :system_user, aliases: '-S',  type: :boolean, default: false,
+ desc: 'Connect to database as system user and request database passwords'
```

Comment:
Actually this option says "don't rely on standard makandra setup". What about calling it `--explicit`? 
```suggestion
option :explicit, aliases: '-e',  type: :boolean, default: false,
 desc: 'Access databases with sudo and ask for database password (MySQL)'
```
Or, we could go with a `default` option that's initially true:
```suggestion
option :default, aliases: '-d',  type: :boolean, default: true,
 desc: 'Rely on makandra standard setup'
```
It can be disabled by passing `--no-default`. That's even prettier in my eyes.

Reply:
I think the `default` option and the description are to vague. I wouldn't be sure, what's encompassed in the default setup. Is maybe postgres default? I wouldn't know that we're talking about authorization for accessing the database.
This option is for people that don't use the standard setup, so imo it should tell explicitly what it does. If I use the default setup, the description doesn't offer any new value. If I don't use the default setup, the description is no help.

So I like the `explicit` suggestion better, but I'm not sure, if it really captures the meaning or implication of what the option is doing. What do you think about calling the option `login`?

```suggestion
option :login, aliases: '-l',  type: :boolean, default: false,
 desc: 'Access databases with sudo and ask for database password (MySQL)'
```

Reply:
We decided to go for `--sudo` with default `false`, but only if `--sudo` enables it and `--no-sudo` keeps it false.

Reply:
We decided to go for `--sudo` with default `false`, but only if `--sudo` enables it and `--no-sudo` keeps it false.

```suggestion
option :sudo, aliases: '-S',  type: :boolean, default: false,
 desc: 'Access databases as root'
```